client/admin/client.go

```diff
@@ -22,14 +22,16 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+// Generates all three generated files in this package:
+//go:generate go run ../../cmd/tools/rpcwrappers -service admin
```

Comment:
This is first time we use `go:generate` for something different than mocks. `make mocks` will execute this also, which is unexpected.

I think this should be leveraged:
```
Go generate accepts one specific flag:

        -run=""
                if non-empty, specifies a regular expression to select
                directives whose full original source text (excluding
                any trailing spaces and final newline) matches the
                expression.
```

Reply:
And dedicated `Makefile` target is needed for this. And it needs to be included in `ci-build`.

Reply:
OK, then don't forget to remove `mocks` target from `Makefile` in that PR too.